Directory.Build.targets

```diff
@@ -0,0 +1,31 @@
+<Project>
+    <Target
+            Name="PublishSamples"
+            Condition="'$(IsSample)' == 'true'">
+        <PropertyGroup>
+            <ArtifactsRoot>$(MSBuildThisFileDirectory)/artifacts/</ArtifactsRoot>
+            <_IsMultiTargetTfm Condition="'$(TargetFrameworks)' != ''and '$(TargetFramework)' == ''">true</_IsMultiTargetTfm>
+            <_IsSingleTargetTfmNoRids Condition="'$(TargetFramework)' != '' and '$(RuntimeIdentifiers)' == ''">true</_IsSingleTargetTfmNoRids>
+            <_IsSingleTargetAndMultiRid Condition="'$(TargetFramework)' != '' and '$(RuntimeIdentifiers)' != ''">true</_IsSingleTargetAndMultiRid>
```

Comment:
The SDK (or MSBuild) could provide helper properties like this to make Conditions easier for authors.